tests/integration/run.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+set -eu
+
+cleanup() {
+        ${DOCKER_BIN}/cluster_down.sh
+}
+
+trap cleanup 0 2 3 6 15
+
+TEST_SOURCE="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+source ${TEST_SOURCE}/../../docker/.env
+
+if [[ ${1:-} == "help" ]]; then
+    python ${TEST_SOURCE}/integration_test.py --help
+    exit 1
+fi
+
+#start cluster
+${DOCKER_BIN}/cluster_up.sh
+
+if [[ $? -ne 0 ]]; then
```

Comment:
` .. cluster_up.sh || exit 1` 
but this is most likely already the case since you have `set -e`

Reply:
removing outright in favor of `-eu`